tests/backends/test_arviz.py

```diff
@@ -213,15 +213,16 @@ def test_predictions_to_idata_new(self, data, eight_schools_params):
 
     def test_posterior_predictive_keep_size(self, data, chains, draws, eight_schools_params):
         with data.model:
-            posterior_predictive = pm.sample_posterior_predictive(
-                data.obj, return_inferencedata=False
-            )
-            inference_data = to_inference_data(
-                trace=data.obj,
-                posterior_predictive=posterior_predictive,
-                coords={"school": np.arange(eight_schools_params["J"])},
-                dims={"theta": ["school"], "eta": ["school"]},
-            )
+            with pytest.warns(UserWarning, match="Incompatible coordinate length"):
```

Comment:
Why were the coords invalid? I don't think they are supposed to be in this test, so maybe the test was faulty?